packages/venia-ui/lib/components/LegacyMiniCart/miniCart.js

```diff
@@ -10,6 +10,9 @@ import Mask from './mask';
 import defaultClasses from './miniCart.module.css';
 import { useStyle } from '../../classify';
 
+/**
+ * @deprecated
```

Comment:
Keep up with the deprecation descriptions, and linking to what it was replaced with 👍